### PR TITLE
Early 21.05 dep update

### PR DESCRIPTION
**Pull request by dannon**
I updated these when checking for adding a new dependency.  There was a resolution conflict so I figured I'd sort it out and get it committed before it bit anyone else.

### CODE REVIEW

**Comment by dannon** [comment]
I would guess that the the jwt-related unit test failure is likely legit; I'll take a look tomorrow and see what changed in the new version.

File: lib/galaxy/dependencies/pipfiles/default/Pipfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ name = "pypi"
 gunicorn = "*"
 lxml = "!=4.2.2"
 NoseHTML = "*"
-pygithub = "*"
+pygithub = "==1.54"
```
Comment (nsoranzo) [inline lib/galaxy/dependencies/pipfiles/default/Pipfile:15]: Are we sure that pygithub 1.54 actually works correctly with pyjwt 2.x ?

Comment (dannon) [reply lib/galaxy/dependencies/pipfiles/default/Pipfile:15]: I'll check -- been distracted for a few.

**Comment by dannon** [comment]
Closing in favor of #11192, which pulls in the changes in 722260d as well.

**Pull request closed without merging**